### PR TITLE
IA-2586 Add workflow that merges all scala-steward PRs into one

### PR DESCRIPTION
**Pull request by Qi77Qi**
Example: https://github.com/DataBiosphere/leonardo/pull/1976

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by rtitle** [comment]
The workflow LGTM, but I'm curious why auto-tests did not run on https://github.com/DataBiosphere/leonardo/pull/1974. Maybe needs a retest?

**Comment by Qi77Qi** [comment]
not sure...it also triggered https://github.com/DataBiosphere/leonardo/pull/1974#issuecomment-810671245... asking in https://broadinstitute.slack.com/archives/CADM7MZ35/p1617203707145900

**Comment by Qi77Qi** [comment]
@rtitle should be good now...see #1976

File: .github/workflows/merge_scalasteward_prs.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        while IFS= read -r pr
+        do
+          echo "$pr"
+          gh pr checkout $pr
+          curBranch=`git branch --show-current`
+          git checkout -B merged-scala-steward-prs
+          git merge -m "merging" $curBranch
+        done <<< `gh pr list | grep scala-steward | cut -d$'\t'  -f 1`
+        git push origin merged-scala-steward-prs
+        gh pr create --title "Bump dependencies" --body "Consolidate scala-steward PRs"
```
Comment (rtitle) [inline .github/workflows/merge_scalasteward_prs.yml]: Should it say `[no ticket] Bump dependencies`?

**Review by rtitle — APPROVED** [review]

**Pull request merged**